.bazelrc

```diff
@@ -19,7 +19,7 @@
 
 build --incompatible_strict_action_env --javacopt='--release 8'
 run --incompatible_strict_action_env
-test --incompatible_strict_action_env
+test --incompatible_strict_action_env --test_env=PATH
```

Comment:
This change allows Bazel to use the `lsof` utility. Perhaps, in the future, we should change it and include `lsof` in our Bazel runfiles.